features/conveyancer_provides_property_address.feature

```diff
@@ -11,14 +11,15 @@ Acceptance Criteria
           a number or a number followed by a letter in the first)
   (us106) Commas are not displayed on the address when borrower views deed
 
-@us85a
+@us85a @wip
 Scenario: No property address provided
-  Given I do not have a property address to provide
+  Given I setup a deed with <1> borrowers
+  And I amend "property_address" to ""
   When I create the deed via the Deed API
   Then a status code of "400" is returned
   And a message for failure is given "Failed validating 'pattern' in schema['properties']['property_address']:"
 
-@us106
+@us106 @wip
 Scenario: Input property address formatted with commas
   Given I have a property address that is formatted with commas
```

Comment:
Any reason why this step was left in and not replaced with the step  'And I amend "property_address" to "something, something, something"'